src/frontend/src/lib/components/icons/IconGlasses.svelte

```diff
@@ -0,0 +1,34 @@
+<!-- source: ISC Lucide - please visit https://lucide.dev/license -->
```

Comment:
Note: we are currently discussing internally how to properly give attribution for ISC Lucide